### PR TITLE
Add suffixes to typos that start with "n"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -34651,45 +34660,59 @@ nealy->nearly, newly,
...
+necessesarily->necessarily
 necessesary->necessary
 necessiate->necessitate
 nechanism->mechanism
+necssarily->necessarily
 necssary->necessary
 ned->need, end,
 nedd->need
 nedded->needed
+nedding->needing
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34714]: Probably `nodding`, perhaps even `wedding` and `bedding`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -35323,12 +35353,42 @@ nodels->models
...
+nofifies->notifies
+nofify->notify
+nofifying->notifying
+nofitication->notification
+nofitications->notifications
+nofitied->notified
+nofities->notifies
 nofity->notify
+nofitying->notifying
+nohing->nothing
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35368]: Perhaps `notching`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -35323,12 +35353,42 @@ nodels->models
...
 nohypen->nohyphen
 noice->noise, nice, notice,
+noification->notification
+noifications->notifications
+noified->notified
+noifies->notifies
+noify->notify
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35375]:
```suggestion
noify->notify, noisy,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -35472,7 +35556,19 @@ normalyy->normally
...
+normlised->normalised
+normlises->normalises
+normlising->normalising
+normlization->normalization
+normlize->normalize
+normlized->normalized
+normlizes->normalizes
+normlizing->normalizing
 normlly->normally
+normls->normals
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35571]: `norms` too?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -35472,7 +35556,19 @@ normalyy->normally
 normalyzation->normalization
 normalyze->normalize
 normalyzed->normalized
+norml->normal
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35559]: `norm` too?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -35621,6 +35731,11 @@ nubmers->numbers
 nucleous->nucleus, nucleolus,
 nucular->nuclear
 nuculear->nuclear
+nueral->neural, numeral,
+nueron->neuron
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35735]: Add `nueronal->neuronal` as a variant of `nueral->neural?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -35650,6 +35765,7 @@ numberss->numbers
 numbert->number
 numbet->number
 numbets->numbers
+numbre->number
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:35768]: Add `numbred->numbered`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:34714]: I'll add `nodding`. I couldn't find the other two typos.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:35368]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**